.infrastructure/41_iam_users.tf

```diff
@@ -0,0 +1,27 @@
+module "iam_user_mauro_dandrea" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-iam.git//modules/iam-user?ref=f37809108f86d8fbdf17f735df734bf4abe69315" # v5.34.0
+  count  = var.environment == "dev" ? 1 : 0
+
+  force_destroy                 = true
+  password_reset_required       = true
+  create_iam_user_login_profile = false
```

Comment:
in my opinion this line should be removed (default value = true) or set as true to allow login